src/likelihoods/negativebinomial.jl

```diff
@@ -51,7 +51,7 @@ julia> var(d) ≈ exp(2.0) * (1 + 3.0)
 true
 ```
 
-[^1] Hilbe, Joseph M. Negative binomial regression. Cambridge University Press, 2011.
+[^Hilbe'11]: Hilbe, Joseph M. Negative binomial regression. Cambridge University Press, 2011.
```

Comment:
@theogf the `:` was missing. But then I thought a named footnote might be nicer anyways.